charts/trino/values.yaml

```diff
@@ -664,6 +664,32 @@ coordinator:
 
   affinity: {}
 
+  topologySpreadConstraints: []
+  # coordinator.topologySpreadConstraints -- Configure [topology spread
+  # constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/)
+  # to control how coordinator pods are spread across your cluster among failure-domains
```

Comment:
There's always a single coordinator pod.

Reply:
I've adjusted the wording. Thank you for the feedback.